README.md

```diff
@@ -60,23 +62,21 @@ module "vpc" {
         subnet-02 = []
     }
 
-    routes = [
-        {
-            name                   = "egress-internet"
+    routes = {
```

Comment:
I don't think we need to change this into a map. We can still maintain the desired behavior by constructing the map (for for_each) internally within the module.

Reply:
sure, makes sense. 

Ill create a local var map with the original default `name` as the key.